test_data.rb

```diff
@@ -34,11 +34,11 @@ def has_rail_id(row)
 
 STATIONS.each do |row|
   if row["same_as"]
-    ALIASES[row["same_as"]] << row
+    (ALIASES[row["same_as"]] || []) << row
```

Comment:
:thinking: we are setting the hash values as empty arrays for all existing stations in lines 18 and 19.

I don't have an impression we have a test covering the wrong ids in referential columns. Would it make sense to add it?

Reply:
It is indeed, thank you